Assets/UI/TabWidgets/Buttons/MainButton/MainButtons/ZoomButton.gd

```diff
@@ -0,0 +1,24 @@
+@tool
```

Comment:
No need for `@tool`. This is only to run scripts in-editor and there's no need for the zoom buttons right now (also it will cause an error when opening TabWidgets as it is accessing `Global` members (singletons are only available in-game)).
```suggestion

```